back/app/controllers/web_api/v1/permissions_controller.rb

```diff
@@ -52,7 +52,11 @@ def serialize(permission)
   end
 
   def permissions_service
-    @permissions_service ||= PermissionsService.new
+    @permissions_service ||= Permissions::PermissionsUpdateService.new
+  end
+
+  def registration_requirements_service
```

Comment:
Maybe it would be nice to keep the names consistent to simplify reading.
```suggestion
  def user_requirements_service
```